runner/operator/access/v1_0_0/msi/__init__.py

```diff
@@ -36,6 +36,7 @@ def get_sample_inputs(self):
         # Get the latest completed runs for the given request ID
         group_id = Run.objects.filter(
             tags__requestId=self.request_id,
+            app__name='access legacy',
```

Comment:
Had to add this line so that if the most recent run for a request ID was any pipeline besides the `access legacy` pipeline, it would find the wrong run